app/controllers/admin/settings/images.js

```diff
@@ -4,8 +4,8 @@ export default Controller.extend({
   actions: {
     saveImages() {
       this.set('isLoading', true);
-      let images = this.get('model');
-      images.save()
+      this.get('model.eventImageSize').save();
```

Comment:
@ritikamotwani the `.then()`, and `.catch()` you have written are handling only the second `save` call, not the first one. If an error occurs while saving `eventImageSize` it won't be handled.